tests/unit/algorithms/classification/test_helper.py

```diff
@@ -25,10 +25,10 @@
 from otx.api.entities.shapes.rectangle import Rectangle
 from otx.api.entities.subset import Subset
 from otx.api.entities.task_environment import TaskEnvironment
+from otx.mpa.utils.config_utils import MPAConfig
 
-DEFAULT_CLS_TEMPLATE = (
-    Path("otx") / "algorithms" / "classification" / "configs" / "mobilenet_v3_large_1_cls_incr" / "template.yaml"
-)
+DEFAULT_CLS_TEMPLATE_DIR = Path("otx") / "algorithms" / "classification" / "configs" / "mobilenet_v3_large_1_cls_incr"
```

Comment:
Can I ask why did you use `mobilenet_v3_large_1_cls_incr` instead of `effnet-b0`? it's just a question :)

Reply:
no specific reason. Maybe I just like mobilenet more than efficientnet. :)